nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/graph-controls/navigation-control/navigation-control.component.scss

```diff
@@ -18,9 +18,12 @@
 div.navigation-control {
     margin-bottom: 2px;
 
-    .fa,
-    .icon {
-        font-size: 18px;
+    .mat-mdc-icon-button {
+        height: 34px !important;
+        width: 34px !important;
+        line-height: 34px;
+        font-size: 17px !important;
+        padding: 0px !important;
```

Comment:
Rather than using !important because this is in a scoped context of the navigation control, we can set new values for the material variables, and it'll only affect items in this context.